src/app/models/contracts-info.ts

```diff
@@ -0,0 +1,9 @@
+export interface ContractsInfo {
+    readonly contracts_version: string;
+    readonly token_network_registry_address: string;
+    readonly secret_registry_address: string;
+    readonly service_registry_address: string;
+    readonly user_deposit_address: string;
+    readonly monitoring_service_address: string;
+    readonly one_to_n_address: string;
```

Comment:
Well that's the data that is returned by the API and we use snake-case there. All the data that comes directly from the API uses snake-case in the WebUI. I don't think it would make sense to transform all the incoming data.